### PR TITLE
add swashes and pyswashes

### PR DESCRIPTION
**Pull request by lrntct**
This PR add the binary package swashes and its python interface pyswashes.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pyswashes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+test:
+  imports:
+    - pyswashes
+  requires:
+    - pytest
+  commands:
+    # error: does not find python
+    # - {{ PYTHON }} -mv pytest -v --pyargs {{ name }}
+    # error: pytest does not find any tests, although they are in the pypi package
+    # - pytest -vv --pyargs {{ name|lower }}
```
Comment (marcelotrevisani) [inline recipes/pyswashes/meta.yaml:37]:
```suggestion
```

File: recipes/swashes/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+
+about:
+  home: https://sourcesup.renater.fr/projects/swashes/
+  license: CECILL-2.0
+  license_family: OTHER
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (marcelotrevisani) [inline recipes/swashes/meta.yaml:27]:
```suggestion
  license_file: LICENSE
```

**Comment by lrntct** [comment]
I believe I attended the comments. Is there something else I can do?

**Comment by lrntct** [comment]
@conda-forge-admin, please ping team

File: recipes/swashes/bld.bat [hunk 3]
```diff
@@ -0,0 +1,4 @@
+cmake.exe CMakeLists.txt
+cmake.exe --build . --config Release
+if not exist "%PREFIX%\" mkdir "%PREFIX%"
+COPY bin\swashes.exe %PREFIX%\
```
Comment (scopatz) [inline recipes/swashes/bld.bat:4]: This file should have exit conditions after each line, see https://conda-forge.org/docs/maintainer/knowledge_base.html#using-cmake

File: recipes/swashes/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  patches:
+    - swashes.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
```
Comment (scopatz) [inline recipes/swashes/meta.yaml:21]: this probably implicitly uses make on unix

File: recipes/swashes/bld.bat (hunk 3 above)
Comment (lrntct) [reply recipes/swashes/bld.bat:6]: Done in 67c93f9

File: recipes/swashes/meta.yaml (hunk 4 above)
Comment (lrntct) [reply recipes/swashes/meta.yaml:21]: Done in 8b6fd97

**Comment by lrntct** [comment]
@conda-forge-admin, please ping team

**Comment by lrntct** [comment]
@conda-forge-admin, please ping team

**Pull request merged**